recipes/sleepecg/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "sleepecg" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fd89d91d5ee5bc1ff640adb68a5b184b52022a987ab730368307a0ee7dbfca5f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.20.0
```

Stick with the oldest possible one here.
```suggestion
    - numpy 1.20.*
```